src/meta/processors/admin/RestoreProcessor.h

```diff
@@ -35,6 +35,10 @@ class RestoreProcessor : public BaseProcessor<cpp2::ExecResp> {
   nebula::cpp2::ErrorCode replaceHostInZone(const HostAddr& ipv4From,
                                             const HostAddr& ipv4To,
                                             bool direct = false);
+
+  nebula::cpp2::ErrorCode replaceHostInMachine(const HostAddr& ipv4From,
```

Comment:
what's the use of the function, why is the default value of direct false?could you add some comment?thank you.

Reply:
I'll remove the direct and add some comments above the processor.